sass/accessibility/_ScreenReader.scss

```diff
@@ -0,0 +1,7 @@
+.coveo-visible-to-screen-reader-only {
+  position: absolute;
+  left: -10000px;
```

Comment:
Why not a simply display: none ? Or visibility: hidden ?

Reply:
If the aria-live element has either display: none or visibility: hidden, the screen-reader will not detect the element and will not read updates. Technically, the height/width = 1px and overflow: hidden are enough to hide the element. The position and left attributes are not needed, but this collection of styles was recommend by Google on one of their pages discussing accessibility. Maybe without position absolute, if there were multiple elements using the class `coveo-visible-to-screen-reader-only`, the elements would start to take noticeable space on the page.

Considering leaving it as-is for now.